docs_src/examples/use_cases/example_indiv_hvp_retain_graph.py

```diff
@@ -0,0 +1,134 @@
+"""Individual Hessian-Vector-Product and retain_graph
+======================================================
+"""
+# %%
+# imports
+import torch
+from torch import nn
+from torch.autograd import grad
+
+from backpack import backpack, extend
+from backpack.extensions import BatchGrad
+from backpack.utils.examples import load_one_batch_mnist
+
+BATCH_SIZE = 32
+DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+torch.manual_seed(0)
+
+
+# %%
+# Let's define a function that calculates individual hessian_vector products.
+# This function was provided by Haonan Wang (github: haonan3) in #213.
+def batch_hvp(model, loss, params_list, batch_grad_list):
+    if len(params_list) != len(batch_grad_list):
+        raise (ValueError("w and v must have the same length."))
+
+    with backpack(retain_graph=True):
+        one_sample_grad_list = grad(
+            loss, params_list, retain_graph=True, create_graph=True
+        )
```

Comment:
I think it would be better to replace this with a `loss = forward(); loss.backward()` which is closer to how BackPACK is supposed to be used.